src/app/containers/CpsRelatedContent/index.jsx

```diff
@@ -1,20 +1,24 @@
 import React, { useContext } from 'react';
 import { arrayOf, shape, number, bool } from 'prop-types';
 import { pathOr } from 'ramda';
-
+import styled from '@emotion/styled';
 import { storyItem } from '#models/propTypes/storyItem';
 import { ServiceContext } from '#contexts/ServiceContext';
 import CpsOnwardJourney from '../CpsOnwardJourney';
 import RelatedContentPromo from './RelatedContentPromo';
 import RelatedContentPromoList from './RelatedContentPromoList';
 
+const StyledCpsOnwardJourney = styled(CpsOnwardJourney)`
+  margin-bottom: 1rem;
```

Comment:
Only thing, can we use a GEL_SPACING value for this? I think 1rem is equal to GEL_SPACING_DBL or GEL_SPACING_QUAD.

Reply:
Yea good shout! Will amend